### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nagarajuv-jfrog**
Updated README with Groovy 4 related content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,57 @@
...
+Artifactory has been upgraded to Groovy 4 starting from version 7.101. This includes several important updates, most notably the promotion plugin (JFrog Supported), updated for compatibility with Groovy 4. However, this upgrade may break compatibility with your custom plugins developed for earlier Groovy versions (Groovy 3 or older). To learn the differences between Groovy 3 and 4, refer to [Release notes for Groovy 4.0
+](https://groovy-lang.org/releasenotes/groovy-4.0.html)
 
-1. **JDK17 Compatibility**: With the introduction of JDK17 support in Artifactory version **7.43** and above, many older user plugins may no longer be compatible. Deprecated plugins are now moved to the [deprecated directory](http://github.com/jfrog/artifactory-user-plugins/tree/master/deprecated-plugins).
+**JFrog Supported Plugins**
 
-Please consider migrating to JFrog Workers for cloud-native and future-proof solutions.
+- **Promotion Plugin**
+  
+    The promotion plugin has been updated to work with Groovy 4. If you're using this plugin in your environment, redeploy it after upgrading to Groovy 4.
```
Comment (yashprit-jfrog) [inline README.md:16]: last line should say upgrading of Artifactory cloud version 7.102 instead of groovy 4. As groovy 4 is part of RT release

Comment (yashprit-jfrog) [reply README.md:16]: @nagarajuv-jfrog we also need to mentioned about deprecated plugin. We have not tested them, if customer is using they need to test them & change accordingly.

Comment (nagarajuv-jfrog) [reply README.md:16]: > 7.102

it's 7.101. right?

Comment (yashprit-jfrog) [reply README.md]: yes

**Pull request merged**